### ISSUE TITLE
Error: undefined method 'load_manifest' for Lucky::AssetHelpers:Module

### ISSUE THREAD

**Issue by pateljoel** [luckyframework/lucky#1584]
**Describe the bug**
I get an error when running `./script/setup`

**To Reproduce**

1. Go through the [Lucky overview guide](https://luckyframework.org/guides/tutorial/overview): 
2. Run [./script/setup](https://luckyframework.org/guides/tutorial/overview#setup-script)
3. Error is shown below:

```
▸ Creating the database
Showing last frame. Use --error-trace for full trace.

In src/app.cr:4:21

 4 | Lucky::AssetHelpers.load_manifest "public/mix-manifest.json"
                         ^------------
Error: undefined method 'load_manifest' for Lucky::AssetHelpers:Module
```

**Expected behavior**
Setup should complete as expected.

**Versions (please complete the following information):**
 - Lucky version (check in shard.lock): 0.28.0
 - Crystal version (`crystal --version`): Crystal 1.1.1 [6d9a1d583] (2021-07-26)
 - OS: Mac OS Catalina

**Comment by jwoertink** [comment]
Hey @pateljoel Thanks for reporting this issue. This is caused by pulling from the master branch of Lucky. If you went through the guide to setup and install Lucky manually (by cloning the CLI repo), then my guess is you missed [this step](https://luckyframework.org/guides/getting-started/installing#3-check-out-the-latest-released-version) which has you checkout the latest version. Can you check and make sure that you're using the latest version of Lucky, and not pulling from the master branch?

**Comment by pateljoel** [comment]
@jwoertink Thanks for the reply, I'll try these steps again and report back.

**Comment by jwoertink** [comment]
I'm closing this since it was related to the version, and it's been fixed in Lucky 0.29
